help/en/releasenotes/current-draft-note.shtml

```diff
@@ -71,7 +71,10 @@
 
         <h4>LocoNet</h4>
             <ul>
-                <li></li>
+                <li>Initial support for LocoNet Multi Sense Long OpCode.  This Long OpCode is used by
```

Comment:
This release note information is more detailed and developer-driven than I prefer.  I prefer to see an end-user view in the release notes, and a developer view in the GIT commit messages.  End-users rarely care about LocoNet OpCodes or message "names", but they do care about _changes to functionality_ that they may or may not see on their layout.

With that view in mind, it would be _my preference_ that the release note message say something more like:
`Provides LocoNet Monitor interpretation of messages generated by the Digikeijs DR5088RC "RailCom" device.`

Reply:
Agreed.  Updated.  Thanks for the feedback.